webapi/src/main/scala/org/knora/webapi/messages/util/CalendarDateUtilV2.scala

```diff
@@ -517,7 +518,8 @@ object CalendarDateRangeV2 {
   def parse(dateStr: String): CalendarDateRangeV2 = {
     // Validate the date string.
     val stringFormatter = StringFormatter.getGeneralInstance
```

Comment:
is this stringformatter instance here still needed?

Reply:
Yes and only because of the constans it uses from SF. I left it there only because I don't have the clear view where all of coasts will leave. I guess this would be easiest to do at the final task and then the instances of SF would be also removed.